tests/test_assets/gx/checkpoint_result_pass.json

```diff
@@ -0,0 +1,428 @@
+{
+    "run_id": {
+        "run_name": null,
+        "run_time": "2024-02-29T17:26:00.099052-07:00"
+    },
+    "run_results": {
+        "ValidationResultIdentifier::genes_biodomains/__null__/20240301T002600.099052Z/default_pandas_datasource-#ephemeral_pandas_asset": {
+            "validation_result": {
+                "success": false,
```

Comment:
Shouldn't this be "true" if this file is for a passing result?

Reply:
For the purposes of these files the only thing that really matters is the "outer" success [field](https://github.com/Sage-Bionetworks/agora-data-tools/blob/5f1fd176e92a149f3a3e9bba9acde7b6cbe14213/tests/test_assets/gx/checkpoint_result_pass.json#L427-L428), but I can see why this is a bit confusing so I will change this.

Reply:
changed in my latest commit